src/app/hooks/useHasRenderedClientSide.tsx

```diff
@@ -0,0 +1,15 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { useEffect, useState } from "react";
+
+export function useHasRendedClientSide() {
```

Comment:
Nit: "Rended" --> "Rendered" (plus the 2&times; other instances where this is imported and used above in `src/app/(proper_react)/(redesign)/(public)/RebrandAnnouncement.tsx`)
```suggestion
export function useHasRenderedClientSide() {
```

Reply:
Thank you! 690b5fccd